acurl/setup.py

```diff
@@ -28,6 +28,5 @@
             "warn.unused_result": True,
         },
     ),
-    setup_requires=["setuptools_scm", "cython"],
```

Comment:
What was setuptools_scm giving us before?

Reply:
`0.1.dev231+g6f2b85e` as we don't currently have/set a version in git, it looks ugly like this.

If we create a git tag, we can use this and it looks nice (e.g. 1.0.0), which is what I suggest we do for mite itself.

Also PyPi doesn't accept this as a version and you can't upload packages with the version looking like this.